core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/GoogleBillingProjectDeleteRunner.scala

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.dsde.rawls.monitor.workspace.runners
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.billing.{BillingProjectLifecycle, BillingRepository}
+import org.broadinstitute.dsde.rawls.dataaccess.{GoogleServicesDAO, SamDAO}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{
+  WorkspaceManagerResourceJobRunner,
+  WorkspaceManagerResourceMonitorRecord
+}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.{
+  Complete,
+  Incomplete,
+  JobStatus
+}
+import org.broadinstitute.dsde.rawls.model.{CreationStatuses, RawlsBillingProjectName}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+class GoogleBillingProjectDeleteRunner(
```

Comment:
This is super simple - most of it is just sanity checking some values. Then it just calls the same code the orchestrator was, before.